nupic/algorithms/KNNClassifier.py

```diff
@@ -1009,14 +1032,13 @@ def leaveOneOutTest(self):
 
 
   def remapCategories(self, mapping):
-    """Change the category indices.
-
-    Used by the Network Builder to keep the category indices in sync with the
-    ImageSensor categoryInfo when the user renames or removes categories.
+    """Change the category indices. Used by the Network Builder to keep the
```

Comment:
PEP-257 recommends a single sentence on the first line, then blank line, then detailed description. So the previous version followed the format at the beginning a little better I think